<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
    <meta content="" name="description"/>
    <meta content="" name="author"/>
    <!-- For iPhone -->
    <meta name="msapplication-TileColor" content="#4076e0">
    <!-- Chrome, Firefox OS and Opera -->
    <meta name="theme-color" content="#4076e0">
    <!-- Windows Phone -->
    <meta name="msapplication-navbutton-color" content="#4076e0">
    <title>Virtual Labs</title>
    <link rel="shortcut icon" href="./assets/images/favicon.ico" />
    <link rel="stylesheet" href="./assets/css/bootstrap.min.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/github-markdown-css/4.0.0/github-markdown.min.css" integrity="sha512-Oy18vBnbSJkXTndr2n6lDMO5NN31UljR8e/ICzVPrGpSud4Gkckb8yUpqhKuUNoE+o9gAb4O/rAxxw1ojyUVzg==" crossorigin="anonymous" />
    <link
      href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Raleway&display=swap"
      rel="stylesheet"
    />
    <link
      rel="stylesheet"
      href="./assets/fonts/font-awesome-4.7.0/css/font-awesome.min.css"
    />
    <link rel="stylesheet" href="./assets/css/github-markdown.min.css" />
    <link rel="stylesheet" href="./assets/css/vlabs-style.css" />
    
    <script src="./assets/js/jquery-3.4.1.slim.min.js"></script>
    <script
      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.5.4/dist/umd/popper.min.js"
      integrity="sha384-q2kxQ16AaE6UbzuKqyBE9/u/KzioAlnx2maXQHiDX9d4/zp8Ok3f+M7DPm+Ib6IU"
      crossorigin="anonymous"
    ></script>
    <script
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta1/dist/js/bootstrap.min.js"
      integrity="sha384-pQQkAEnwaBkjpqZ8RU1fF1AKtTcHJwFl3pblpTlHXybJjHpMYo79HY3hIi4NKxyj"
      crossorigin="anonymous"
    ></script>
    <style>
      h1 {
      font-size: 1.8rem;
      }
      h2 {
      font-size: 1.4rem;
      }
      h3 {
      font-size: 1.2rem;
      }

      h2, h3 {
      font-weight: 800;
      }
    </style>
  </head>
  <body class="p-0 container-fluid vlabs-page d-flex flex-column justify-content-between">
    <header class="vlabs-header sticky-top bg-white">
      <nav class="p-0 navbar navbar-light d-flex align-items-stretch">
        
        <button class="navbar-toggler px-4"
        	    type="button" onclick="toggle()">
          <span class="navbar-toggler-icon"></span>
        </button>
    
        <div class="d-flex justify-content-center">        
          <a href="http://vlab.co.in" class="navbar-brand">
            <img src="./assets/images/vlabs-color-small-moe.jpg" alt="vlabs logo">
          </a>
        </div>
        
        <div id="headerNavbar" class="border-top flex-grow-1 d-flex align-items-center">
          <ul class="navbar-nav ml-auto text-center d-flex flex-md-row">
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in">HOME</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/participating-institutes">PARTNERS</a>
            </li>
            <li class="nav-item px-2 m-1">
              <a href="http://vlab.co.in/contact-us">CONTACT</a>
            </li>
          </ul>
        </div>
      </nav>
    </header>
    <div class="container-fluid flex-fill d-flex flex-column vlabs-page-main">
      <div class="row flex-grow-1 px-4 d-flex flex-nowrap">
        <div class="nav flex-column sidebar flex-shrink-0" id="menu">
            <div>
              <div class="d-flex">
                <a href="index.html" class="p-2 ">
                  Aim
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="theory.html" class="p-2  current-item ">
                  Theory
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="pretest.html" class="p-2 ">
                  Pretest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="procedure.html" class="p-2 ">
                  Procedure
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="simulation.html" class="p-2 ">
                  Simulation
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="posttest.html" class="p-2 ">
                  Posttest
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="references.html" class="p-2 ">
                  References
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
            <div>
              <div class="d-flex">
                <a href="feedback.html" class="p-2 ">
                  Feedback
                </a>
                
              </div>
              <div id="t-"
                
                   class="collapse" 
                >
                <div class="px-1 d-flex flex-column tasks">
                </div>
              </div>
            </div>
        </div>        <div class="vlabs-page-content px-4 pb-4 flex-grow-1 markdown-body">
          <div class="text-center">
            <h2 id="uniprot-protein-sequence-retrieval-in-r">UniProt Protein Sequence Retrieval in R</h2>
        
          </div>
            <p>Prediction of protein sequences y high-throughput genome sequencing expands the large-scale need of proteomics. There has been a need for understanding protein sequence repository and systematic analysis of protein annotation and standardization of protein data from the existing resources. It has been noted that new data types have been introduced with high-throughput technologies in proteomics and genomics for providing opportunities in biomedical sciences and life sciences domain. Identifying experimental characterizations of proteins and integrating the knowledge with annotation approaches were of keen interest in bioinformatics applications. Such data integration method provides diverse information from different resources providing a comprehensive overview of the protein data to the scientific community. In a broader sense, the ability for storing and interconnecting the available protein data information is noted as crucial in modern biological research applications.</p>
        <p>The Universal Protein Resource (UniProt) is freely accessible central resource that provides stable and comprehensive details on protein sequences and functional annotation. It is of primarily important in scientific community as the protein sequences were fully classified accurately with extensive cross-referencing and maximum querying interfaces thereby providing interconnection with information from pool of disparate resources.  UniProt Consortium is formed in 2002, which is collaboration between European Bioinformatics Institute (EBI), the Protein Information Resource (PIR) and the Swiss Institute of Bioinformatics (SIB). By means of computational analysis, and sequence archiving, this consortium functions in manual curation  of protein sequences of interest and develop a user-friendly web platform that cross-references to other databases providing value added information of protein sequences thereby maintaining a high quality database. </p>
        <p>&nbsp;</p>
        <p>The main applications of Uniprot are</p>
        <ol>
        <li>Identifying protein sequences and functional information for research application in life science domains. </li>
        <li>Provides detailed information on biological function of proteins. </li>
        <li>Provides evolutionary relationships of protein sequence of interest. </li>
        <li>Studying the functional properties of proteins in detail. </li>
        <li>Understanding the biological properties of a specific protein that were involved in a biological reaction.  </li>
        <li>Understanding the translation and post-translation modifications in proteins. </li>
        <li>Provides overview of a protein interaction with other biological molecules. </li>
        <li>Helps in localizing the presence of a specific protein in cells or higher organisms.</li>
        </ol>
        <p>&nbsp;</p>
        <p>UniProt has four major components with optimized usage functions. They are </p>
        <ol>
        <li>the UniProt Knowledgebase (UniProtKB)</li>
        <li>the UniProt Reference Clusters  (UniRef)</li>
        <li>the UniProt Archive (UniParc)</li>
        <li>the UniProt Metagenomic and Environmental Sequences database (UniMES)</li>
        </ol>
        <p>&nbsp;</p>
        <p>UniProt Knowledgebase is divided into two sections UniProtKB/Swiss-Prot and UniProtKB/TrEMBL, where UniProtKB/Swiss-Prot is manually curated and UniProtKB/TrEMBL is maintained automatically. The annotations were accurately performed by biologists having specific expertise in protein sequence annotation. UniProtKB is the central access point for extensive curated protein information providing information of functions of the selected protein, specific enzymes and enzyme-specific reactions, active domains, post-translational modifications, specific expression of protein in sub-cellular regions, formation of isoforms, genetic variants associated with a disease and related abnormalities. UniProtKB/TrEMBL followed general rules such as Spearmint rules, HAMAP family rules and PIRSF classification-based name rules for automatic annotation and classification.  It included coding sequences translational sequences and PDB structure data of amino acids with defined priorities.</p>
        <p>&nbsp;</p>
        <p>UniRef have been applied in genome annotation, system biology applications, understanding structural genomics of protein sequences, classification of protein family, understanding phylogenic relationships and mass spectrometry calculations. (UniParc) is a repository that provides historical aspects of protein sequences. It records information without annotation. UniMES has data related to metagenomics with reference to Global Ocean Sampling Expedition (GOS) and International Nucleotide Sequence Databases (INSDC). Gathered information was available in ftp site in FASTA format. </p>
        <p>&nbsp;</p>
        <p><strong>UniProt Protein Sequence Retrieval in R</strong></p>
        <p>Like NCBI sequence Uniprot also has RefSeq but the manually annoted information quality in UniProt is greater than that in RefSeq. Accesion numbers were assigned to individual protein sequences in Uniprot database. Another method to retrieve a UniProt protein sequence is by the use of  the SeqinR package to query the ACNU sub-database “swissprot”, which contains protein sequences from UniProt.  Use the query () function from SeqinR to query protein sequence.
        UniProt data were downloaded from <a href="http://www.uniprot.org">www.uniprot.org</a> </p>
        <p>&nbsp;</p>
        <p>Example</p>
        <ol>
        <li>Retrieve the protein sequence for chorismate lyase protein in Mycobacterium leprae (Uniprot accession Q9CD83) and Mycobacterium ulcerans (Uniprot Accession A0PQ23).</li>
        <li>Save the sequence date in FASTA format.</li>
        <li>Save the sequence data in My documents folder of personal computer.</li>
        </ol>
        
        </div>
      </div>
    </div>
    <footer class="text-light pt-2 vlabs-footer d-flex flex-column">
      <div class="row px-5">
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Community Links</span>
          <a href="http://www.sakshat.ac.in/" class="text-light">Sakshat Portal</a>
          <a href="http://outreach.vlabs.ac.in/" class="text-light">Outreach Portal</a>
          <a href="http://vlab.co.in/faq" class="text-light">FAQ: Virtual Labs</a>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Contact Us</span>
          <span> Phone: General Information: 011-26582050 </span>
          <span> Email: support@vlabs.ac.in </span>
        </div>
        <div class="col d-flex flex-column">
          <span class="font-weight-bold vlabs-footer-sect-name pb-2 mb-3">Follow Us</span>
          <div class="social-links">
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #55acee;" href="https://twitter.com/TheVirtualLabs">
    	  <i class="fa fa-twitter"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #3b5998;" href="https://www.facebook.com/Virtual-Labs-IIT-Delhi-301510159983871/">
    	  <i class="fa fa-facebook"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #e52d27;" href="https://www.youtube.com/watch?v=asxRaOgk6a0">
    	  <i class="fa fa-youtube"></i>
    	</a>
    	<a class="p-2 mt-1 mr-2 d-inline-flex justify-content-center align-items-center" style="background: #2867B2;" href="https://in.linkedin.com/in/virtual-labs-008ba9136">
    	  <i class="fa fa-linkedin"></i>
    	</a>
          </div>
        </div>
      </div>
      <div class="m-0 py-2 text-center" style='font-family: "Open Sans", sans-serif; background: #212121;'>
        <a class="text-primary font-weight-bold" href="https://www.gnu.org/licenses/agpl-3.0.en.html"> AGPL 3.0 </a>
        &nbsp;&amp;&nbsp;
        <a class="text-primary font-weight-bold" href="https://creativecommons.org/licenses/by-nc-sa/4.0/"> Creative Commons (CC BY-NC-SA 4.0) </a>
      </div>
    </footer>

      <script src="./assets/js/toggleSidebar.js"></script>
      
  </body>
</html>
